phpunit.xml.dist

```diff
@@ -14,9 +14,6 @@
     <include>
       <directory suffix=".php">src</directory>
     </include>
-    <report>
```

Comment:
The main point of this PR is to get rid of this line, which causes PhpUnit to throw a warning and other programs (like paratest) to fatally error. It's better to run coverage independently from automated tests.